stable/storageos-operator/templates/cleanup.yaml

```diff
@@ -114,14 +114,17 @@ spec:
     spec:
       serviceAccountName: storageos-cleanup-sa
       containers:
-      - name: "statefulset-{{ .name }}-cleanup"
+      - name: "storageos-{{ .name }}-cleanup"
         image: bitnami/kubectl:1.14.1
         command:
           - kubectl
+          - -n
+          - {{ $.Values.cluster.namespace }}
           - delete
           {{- range .command }}
           - {{ . | quote }}
           {{- end }}
+          - --ignore-not-found=true
```

Comment:
Avoid failure if the resource is not found.

Reply:
:+1: Does this need to be in the next Rancher PR?

Reply:
Yes, it'll also help not fail when we try to delete CSI deployment/statefulset and one of them is not found. We need this.

Reply:
added in Rancher PR